torch/_torch_docs.py

```diff
@@ -3241,6 +3241,34 @@ def merge_dicts(*dicts):
     tensor([ 0.0000,  0.5000, -0.2000])
 """)
 
+add_docstr(torch.frexp,
+           r"""
+frexp(input, *, out=None) -> (Tensor mantissa, Tensor exponent)
+
+Decomposes :attr:`input` into mantissa and exponent tensors
+such that :math:`\text{input} = \text{mantissa} \times 2^{\text{exponent}}`.
+
+The range of mantissa is the open interval(-1, 1).
+
+.. note:: This function requires inputs to be floating-point dtypes.
```

Comment:
Instead of this note, add a "Supports..." statement like torch.sub() has (https://pytorch.org/docs/master/generated/torch.sub.html#torch.sub):

"Supports float inputs."